script/magic_numbers/Decimal_Division_Generator.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Random;
+import java.math.BigDecimal;
+import java.math.BigInteger; 
+public class Decimal_Division_Generator 
+{ 
+	public static void main(String[] args) 
+	{
+
+		int no_of_test_cases = 1000000;
+
+		Random rand = new Random(); //instance of random class
+        //generate random values from 0-10
+      	int upperbound = 10;
+      	
+
+      	for(int i = 0; i < no_of_test_cases; i++) {
```

Comment:
could you please run intellij's java autoformat or something similar on this and other java files?

Reply:
resolved